ui/apps/app-center/src/extensions/install-modal.tsx

```diff
@@ -103,7 +102,7 @@ const IntegrationInstallModal: React.FC<RootExtensionProps> = props => {
       )}
       errorInfoLabel={t('Please check your network connection and try again.')}
       errorSubInfoLabel={t('Thank you!')}
-      errorSubtitleLabel={t('could not be installed in {{worldName}}.', {
+      errorSubtitleLabel={t('{{integrationName}} could not be installed in {{worldName}}.', {
```

Comment:
you will need to pass this variable to the second param of tFunc, just like it's with worldName